web/index.html

```diff
@@ -104,37 +104,7 @@
                       <link type="text/css" rel="Stylesheet" href="pages/css/wv.document.css">
 
 
-    <!-- External libraries =============================================== -->
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/babel-polyfill/dist/polyfill.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="ext/node_modules/babel-polyfill/dist/polyfill.min.js"></script> -->
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/react/dist/react.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="ext/node_modules/react/dist/react.min.js"></script> -->
-
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/react-dom/dist/react-dom.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="ext/node_modules/react-dom/dist/react-dom.min.js"></script> -->
-
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/worldview-components/browser/wvc.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="ext/node_modules/worldview-components/browser/wvc.min.js?v=@BUILD_NONCE@"></script> -->
-
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/lodash/lodash.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="ext/node_modules/lodash/lodash.min.js"></script> -->
-
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/bluebird/js/browser/bluebird.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="ext/node_modules/bluebird/js/browser/bluebird.min.js"></script> -->
-
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/promise-queue/lib/index.js"></script>
-
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/openlayers/dist/ol-debug.js"></script>
-    <!-- link.dev -->       <script type="text/javascript" src="js/ol.wv/ol.wv.lookupimagetile.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="js/ol.js"></script> -->
-
-    <!-- link.dev  -->      <script type="text/javascript" src="ext/node_modules/gifshot/build/custom/gifshot.custom.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="ext/node_modules/gifshot/build/custom/gifshot.custom.min.js"></script> -->
-
-    <!-- link.dev -->       <script type="text/javascript" src="ext/node_modules/clipboard/dist/clipboard.js"></script>
-    <!-- link.prod --> <!-- <script type="text/javascript" src="ext/node_modules/clipboard/dist/clipboard.min.js"></script> -->
-
-
+  
     <!-- link.dev  -->      <script type="text/javascript" src="ext/main/jquery-2.1.4/jquery.js"></script>
```

Comment:
Why are we leaving the jQuery scripts here?

Reply:
We aren't. We haven't gotten to this yet. This PR is not ready for review. Still work to be done.